proto/loop.proto

```diff
@@ -15,25 +17,15 @@ service SwapClient {
     point onwards, progress can be tracked via the SwapStatus stream that is
     returned from Monitor().
     */
-    rpc LoopOut (LoopOutRequest) returns (SwapResponse) {
-        option (google.api.http) = {
-            post: "/v1/loop/out"
-            body: "*"
-        };
-    }
+    rpc LoopOut (LoopOutRequest) returns (SwapResponse);
```

Comment:
I am curious. Where are rest paths defined if they are no longer in the proto file?

Reply:
They were extracted into their own yaml file. The advantage is that we don't need to import the google annotations protos anymore.